### DISCUSSION TITLE
Game (ARK) does not install

### DISCUSSION THREAD

**Discussion by Pabxy** [Heroic-Games-Launcher/HeroicGamesLauncher#753] [Q&A]
The ark instalation start but after a few seconds stops. How can i solve this?

**Comment by rjohnson4595** [comment]
I am having same issue. Is there a work around for this yet?

**Comment by flavioislima** [comment]
which platform? we need more info than just saying the game does not install.

**Reply by rjohnson4595** [reply]
Well Epic Games platform

Through Heroic Launcher

Game does install but when going to run it asks to install battleye and then it will load a black screen like its loading the game then crash. 

Ive tried running without battleye and other launch options with ARK to no avail

Edit: 
Running Manjaro Gnome Linux on 2 machines, same issues,

1 machine: i5-10gen, GTX 960 2GB, 16GB DDR4 2666MHZ

2nd Machine: 2020 Asus G14 Zephyrus. Ryzen 9 4900HS, RTX2060, 16GB DDR4 3200MHZ

**Reply by flavioislima** [reply]
Ok so the game installs. 
On linux you will need to launch the version without Battleeye, so you need to go to the Game Page, clicking on the card, and you will see the launch options there. Just select the option without the Anti-cheat.

**Reply by rjohnson4595** [reply]
I have. As i stated ive tried launching without battleye and even the other launch options in the game page/settings.

**Reply by rjohnson4595** [reply]
On my Optiplex ( i5-10th gen desktop) i have heroic installed via Flatpak and have gave it access to install in my home directory. 

On my Asus G14 Heroic is installed via AUR

Both have same result. Running without battleye options.

**Reply by rjohnson4595** [reply]
When i get home later i can provide video, screentshots, and logs if needed
